backend/build.gradle

```diff
@@ -129,6 +129,7 @@ dependencies {
     testImplementation "com.github.tomakehurst:wiremock-jre8:2.35.0"
 
     implementation 'ca.uhn.hapi.fhir:hapi-fhir-structures-r4:6.2.4'
+    implementation 'ca.uhn.hapi.fhir:org.hl7.fhir.utilities:5.6.92' //TODO: Remove when hapi-fhir-structures-r4 is upgraded to invulnerable version. See https://security.snyk.io/vuln/SNYK-JAVA-CAUHNHAPIFHIR-3245104.
```

Comment:
Based on this [comment](https://github.com/hapifhir/org.hl7.fhir.core/issues/1082#issuecomment-1400975407), I'm not sure if that latest version is actually sufficient for the fix. I'm sure this one is better than the last one though!